### PR TITLE
optimize: remove unnecessary log outputs and unify the log output path

### PR DESCRIPTION
**Pull request by a364176773**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
image
顺便调整xss最小值，否则无法运行(可能跟jdk版本有关，512k我在linux上跑可以，mac提示不行)

### CODE REVIEW

**Review by xingfudeshi — APPROVED** [review]
LGTM

File: server/src/main/resources/application.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ spring:
 logging:
   config: classpath:logback-spring.xml
   file:
-    path: ${user.home}/logs/seata
+    path: ${BASEDIR:${user.home}}/logs
```
Comment (slievrly) [inline server/src/main/resources/application.yml:25]: Why change the log directory?

File: distribution/bin/seata-server.sh [hunk 2]
```diff
@@ -23,6 +39,6 @@ CMD_LINE_ARGS=$@
 echo "Affected JVM parameters:$JAVA_OPT"
 
 # start
-echo "$JAVACMD ${JAVA_OPT} ${CMD_LINE_ARGS}" > ${BASEDIR}/logs/start.out 2>&1 &
-nohup $JAVACMD ${JAVA_OPT} ${CMD_LINE_ARGS} >> ${BASEDIR}/logs/start.out 2>&1 &
-echo "seata-server is starting, you can check the ${BASEDIR}/logs/start.out"
+echo "$JAVACMD ${JAVA_OPT} ${CMD_LINE_ARGS}" > /dev/null 2>&1 &
+nohup $JAVACMD ${JAVA_OPT} ${CMD_LINE_ARGS} >> /dev/null 2>&1 &
+echo "seata-server is starting, you can check the ${BASEDIR}/logs *.log"
```
Comment (slievrly) [inline distribution/bin/seata-server.sh:44]: Log paths  are inconsistent.

File: server/src/main/resources/application.example.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ spring:
 logging:
   config: classpath:logback-spring.xml
   file:
-    path: ${user.home}/logs/seata
+    path: ${log.home:${user.home}/logs/seata}
```
Comment (slievrly) [inline server/src/main/resources/application.example.yml:25]: `log.home`
Where was it defined ?

File: distribution/bin/seata-server.sh (hunk 2 above)
Comment (a364176773) [reply distribution/bin/seata-server.sh:44]: image

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**